Makefile

```diff
@@ -72,6 +72,12 @@ test-tablesnap-upload-py2.7: docker-build
 test-tablesnap-upload-py3.6: docker-build
 	docker run -it migrate2to3:v1 ./test_tablesnap.sh python3 $(DIRECTORY) $(TIMEOUT_A) $(TIMEOUT_B) $(AWS_ACC_KEY_ID) $(AWS_SEC_ACC_KEY) $(BUCKET) $(HARDLINKS_PATH)
 
+test-tableslurp-upload-py2.7: docker-build
```

Comment:
Trivial: this target should be called `test-tableslurp-download-py2.7`, since `tableslurp` downloads files from S3. Similarly with the `test-tableslurp-upload-py3.6` target.

Reply:
Right, it is a download. Changed.

Reply:
Targets renamed as requested.